Rakefile

```diff
@@ -1,5 +1,10 @@
+require_relative 'dependaseal'
 require 'rspec/core/rake_task'
```

Comment:
You probably won't have RSpec loaded when running on Heroku so this is likely to fail.